### PR TITLE
Merge to class repo

### PR DESCRIPTION
**Pull request by dcuffari**
I've finished cleaning my repo. All I need to add is a finial density plot for my visualization examples folder.

### CODE REVIEW

File: cluster/cluster.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    roar = "I am a cluster!\n"
+    def __init__(self, partnum, clusnum, D, M):
+        """initilizes the cluster object with the following parameters:
+        
+        param particle_cout: the number of particles in the simulation.
+        param particle_dimensions: the number of dimensions.
+        param cluster_count: the number of particle clusters.
+        param cluster_dimensions: the number of dimensions
+        param particle_positions: the x,y,z position of the particles
+        param particle_velocities:
```
Comment (ECGonzales) [inline cluster/cluster.py]: Where is the rest of the definition for these?

Comment (ECGonzales) [reply cluster/cluster.py]: see two below as well

**Review by ECGonzales — DISMISSED** [review]
Remove .pyc files and check pep-8 using pycharm

File: cluster/cluster.py (hunk 1 above)
Comment (dcuffari) [reply cluster/cluster.py]: I added the definitions.

**Comment by dcuffari** [comment]
All pyc files are gone and I checked pep8 using pycharm.

**Review by anisak000 — APPROVED** [review]
Having gone through this code, I can see that it is very clean and concise. I also like that you used different fonts in your headings. Your repo is very well organized and clearly reflects the work you put into it. One thing I notice is that you don't have very many inline comments in the files including book.py, booklimit.py, and paraBook.py. But, I think that most of it it is self-explanatory so it's okay the way it is. I also like how clearly your density plot visualizations communicate information. One thing you might do is to give axes labels and title to scatter_1000_100data.png. But, I think, according to astronomy convention, title may not even be necessary. Overall, very well done.

**Comment by kelle** [comment]
well done!

**Pull request merged**